### ISSUE TITLE
Please add support for mongo URI as address source

### ISSUE THREAD

**Issue by virl** [jeromelebel/MongoHub-Mac#108]
Both for entering connection address and for retrieving resulting uri address.

**Comment by ilanbiala** [comment]
@jeromelebel not too clear, it should really just be one field OR the x fields instead. Can we set that up? When pasting the full URI into the one box, it autopopulates the other boxes.

**Comment by jeromelebel** [comment]
yes, this could be improved.

**Comment by ilanbiala** [comment]
Right now I'm getting an error that my connection isn't connecting. While we are working on this, can you post an example of what can go in the address field?

**Comment by jeromelebel** [comment]
I don't understand. If you have a problem to connect to a server, it has nothing to do with this issue.

To use your mongodb:// URI, you have 2 choices:
- click on the link, MongoHub should open with a connection panel, ready to be saved
- open MongoHub, and in the first window (with all the connections), just paste the URI

**Comment by ilanbiala** [comment]
pasting the URL was not obvious **at all**. Can you make it more obvious and support pasting that same URL into a single field in the new connection window?

**Comment by jeromelebel** [comment]
Yes I already said it twice: this can be improved. This is why this issue is not closed yet.

**Comment by Kikobeats** [comment]
+1
